android app crash: fatal realmexception with primary key constraint violation

```
FATAL EXCEPTION: main
Process: org.ole.planet.myplanet, PID: 30966
io.realm.exceptions.RealmException: Async transaction failed
at io.realm.Realm$1$2.run(Realm.java:1801)
at android.os.Handler.handleCallback(Handler.java:938)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:223)
at android.app.ActivityThread.main(ActivityThread.java:7741)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:966)
Caused by: io.realm.exceptions.RealmException: Primary key field '_id' cannot be changed after object was created.
at io.realm.org_ole_planet_myplanet_model_RealmCourseActivityRealmProxy.realmSet$_id(org_ole_planet_myplanet_model_RealmCourseActivityRealmProxy.java:135)
at org.ole.planet.myplanet.model.RealmCourseActivity.set_id(RealmCourseActivity.kt:42)
at org.ole.planet.myplanet.service.UploadManager.lambda$uploadCourseActivities$21(UploadManager.java:578)
at org.ole.planet.myplanet.service.UploadManager$$ExternalSyntheticLambda10.execute(Unknown Source:2)
at io.realm.Realm$1.run(Realm.java:1745)
at io.realm.internal.async.BgPriorityRunnable.run(BgPriorityRunnable.java:34)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.lang.Thread.run(Thread.java:923)
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -9,8 +9,8 @@ android {
         applicationId "org.ole.planet.myplanet"
         minSdkVersion 21
         targetSdkVersion 34
-        versionCode 1175
-        versionName "0.11.75"
+        versionCode 1178
+        versionName "0.11.78"
         ndkVersion '21.3.6528147'
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true

--- a/app/src/main/java/org/ole/planet/myplanet/model/RealmCourseActivity.kt
+++ b/app/src/main/java/org/ole/planet/myplanet/model/RealmCourseActivity.kt
@@ -10,6 +10,7 @@ import java.util.UUID
 
 open class RealmCourseActivity : RealmObject() {
     @PrimaryKey
+    private var id: String? = null
     private var _id: String? = null
     @JvmField
     var createdOn: String? = null

